cylc/flow/task_pool.py

```diff
@@ -1997,13 +2061,14 @@ def set_prereqs_and_outputs(
                 continue
 
             if prereqs:
-                set_all, valid_prereqs = (
+                set_all, valid_prereqs, valid_xtrigs = (
```

Comment:
You will almost certainly need to review the docstring for `set_prereqs_and_outputs`.